packages/quantic/force-app/main/default/lwc/quanticNotifications/quanticNotifications.css

```diff
@@ -0,0 +1,7 @@
+.notification {
+  background-color: white;
```

Comment:
why use the design token color code for the border color of the toast and not `$color-background-alt` for the background color?